src/helpers/validateAngularVersion.js

```diff
@@ -7,9 +7,16 @@ const { satisfies } = require('semver')
  * @returns {Promise<boolean>}
  */
 const validateAngularVersion = async function (root) {
-  // eslint-disable-next-line n/no-missing-require
-  const packagePath = require.resolve('@angular/core/package.json', { paths: [root] })
-  if (!packagePath) {
+  let packagePath
+  try {
+    // eslint-disable-next-line n/no-missing-require
+    packagePath = require.resolve('@angular/core/package.json', { paths: [root] })
+    if (!packagePath) {
+      console.warn('This site does not seem to be using Angular.')
```

Comment:
I don't think this will ever be reached - it will either resolve or throw

Reply:
Ah right! Thanks, I was blindly wrapping this on a try catch that I missed that's not needed 👍